### PR TITLE
reduce slider chunkiness

### PR DESCRIPTION
**Pull request by ab7pa**

### CODE REVIEW

**Comment by ab7pa** [comment]
Added vertical-align for the label.

File: files/www/cgi-bin/advancedconfig [hunk 1]
```diff
@@ -510,8 +510,8 @@ html.print([[
 .switch {
     position: relative;
     display: inline-block;
-    width: 60px;
-    height: 34px;
+    width: 48px;
+    height: 20px;
```
Comment (aanon4) [inline files/www/cgi-bin/advancedconfig:514]: Add the "vertical-align:middle" bit here instead of inline it

**Review by aanon4 — APPROVED** [review]
Looks good

**Review by ab7pa — COMMENTED** [review]
Done

**Pull request merged**